scripts/echo360.js

```diff
@@ -283,10 +283,7 @@ H5P.VideoEchoVideo = (() => {
         queryString += 'deactivateSound=true&';
       }
       if (options.startAt) {
-        // Round to 3 decimal places to avoid floating point errors server-side
-        options.startAt = Math.round(options.startAt * 1000) / 1000;
-        // Implicit conversion to millis
-        queryString += `startTimeMillis=${options.startAt * 1000}&`;
+        queryString += `startTimeMillis=${Math.round(options.startAt * 1000)}&`;
```

Comment:
Both
```
        options.startAt = Math.round(options.startAt * 1000) / 1000;
        queryString += `startTimeMillis=${options.startAt * 1000}&`;
```
and
```
queryString += `startTimeMillis=${Math.round(options.startAt * 1000)}&`;
```
produce the same `startTimeMillis` result so this change will likely not solve the problem.